timemachine/lib/potentials.py

```diff
@@ -1,7 +1,6 @@
 from typing import List
 
 import numpy as np
-from numpy.typing import NDArray
```

Comment:
Out of curiosity, whats the difference between using NDArray for typing and using np.ndarray for typing? Had assumed the one from the typing module was preferred